hibernate-core/src/main/java/org/hibernate/boot/model/internal/AnnotationBinder.java

```diff
@@ -371,6 +371,12 @@ private static void bindNamedStoredProcedureQuery(
 		}
 	}
 
+	public static void preBindClass(
```

Comment:
Why not just call this `bindFilterDefs` as well?

Reply:
ok 😄 updated the name.